linode/manager
# fix: [M3-8142] - Alignment Regression Of The IP Address Inside Node Pools Table

## Description 📝
Fixes the the vertical alignment of the IP Address inside of the Node Pools table. 

## Changes  🔄
List any change relevant to the reviewer.
- For testing purposes, I disabled the `background-color: transparent` css property on the `copy` icon in order to show the cause of the visual bug.
- ~~Updated the row width percentage values, allowing the `IP Address` column more room since some addresses will cause the text to wrap. The `Status` column does not need as much room and the values in these change maintain a similar proportion as before.~~
- Pass the `noWrap` prop set to `true` on the `<TableCell>` for the IP Address column to prevent longer addresses from wrapping the `copy` icon and resulting in vertical alignment discrepancies. 

## Preview 📷
| Before  | After   |
| ------- | ------- |
| <video src="https://github.com/linode/manager/assets/119514965/b55cb5bf-018a-4a17-a537-599f8ca20d65" /> | <video src="https://github.com/linode/manager/assets/119514965/10ac1227-7686-43dc-8977-9c6d904eb2de" /> |

## How to test 🧪
- Open the web browser developer tools and modify the viewport width as shown in the `Preview` video to ensure consistent results. 

### Verification steps
(How to verify changes)
- Verify that as you adjust the viewport width, the IP Address which are wider than normal do not cause the copy icon to wrap to the next line (causing the IP Address to shift vertically).

## As an Author I have considered 🤔
*Check all that apply*
